pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -17,19 +17,27 @@
 }:
 
 let
-  version = "1.13.0";
+  version = "1.14.0";
 
   src = fetchFromGitHub {
     owner = "paperless-ngx";
     repo = "paperless-ngx";
     rev = "refs/tags/v${version}";
-    hash = "sha256-aIJWEZD98tjfNDQjQfxRR1kOJ4P/fxZP8sw1dKy7apw=";
+    hash = "sha256-O1Miy0GV34YbE1UrLUWZsatpWyfzFLGvX6fQUJnwbuE=";
   };
 
   # Use specific package versions required by paperless-ngx
   python = python3.override {
     packageOverrides = self: super: {
-      django = super.django_4;
+      django = super.django_4.overridePythonAttrs (oldAttrs: rec {
+        version = "4.1.8";
+        src = oldAttrs.src.override {
+          inherit version;
+          hash = "sha256-QyU/S0osOhaUzVLTUXhHJxk0yE29Bgyg6CqfkOVp2rM=";
+        };
+
+        doCheck = false;
+      });
```

Comment:
Any problems with Django 4.2?

Reply:
Paperless-ngx itself explicitly uses `django~=4.1`: 
https://github.com/paperless-ngx/paperless-ngx/blob/405769dc976942305533d766afc6a0a872b6cf45/Pipfile#L13

I've just ran it with django 4.2, that seems to have worked out (both building and tests). So I guess we can run it with 4.2 too.